src/searchbar.cpp

```diff
@@ -87,8 +85,11 @@ void SearchBar::on_currentTitleChanged(const QString& title)
 
 void SearchBar::focusInEvent( QFocusEvent* event)
 {
-    if (event->reason() == Qt::MouseFocusReason)
+    if (event->reason() == Qt::MouseFocusReason) {
         clear();
+        connect(&m_completer, static_cast<void(QCompleter::*)(const QModelIndex &)>(&QCompleter::activated),
```

Comment:
Why changing the cast ?

Reply:
In the Qt documentation, it's the cast they used
![image](https://user-images.githubusercontent.com/47149116/60010485-30c57680-9678-11e9-8669-508d40ff106a.png)

Reply:
There are different version of connect.
The one used in the documentation takes a function (https://doc.qt.io/qt-5/qobject.html#connect-1)
I prefer to use the one that use a "PointerToMemberFunction" (https://doc.qt.io/qt-5/qobject.html#connect-3). I find the code easier to read.
I'm not sure of how we should strictly follow the note (ie what happens if we don't use the function pointer syntax ?) 
What is the documentation you use ? https://doc.qt.io/qt-5/qcompleter.html#activated-1 gives another example.

Reply:
Indeed it was the qt 4.8